pkg/util/kudo/versions.go

```diff
@@ -0,0 +1,100 @@
+package kudo
+
+import (
+	"sort"
+
+	"github.com/Masterminds/semver"
```

Comment:
Any reason to use the V1 of this package instead of V3?
```suggestion
	"github.com/Masterminds/semver/v3"
```

Reply:
Nope, no reason apart from IntelliJ-Auto-Import ;)